test/thermodynamics/pressure_of_ideal_gas_from_height_and_temperature_test.py

```diff
@@ -0,0 +1,81 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to
+)
+
+from symplyphysics.laws.thermodynamics import pressure_of_ideal_gas_from_height_and_temperature as pressure_of_ideal_gas
+
+
+@fixture(name="test_args")
```

Comment:
Why did you remove the link to the example?

Reply:
Accidentally